src/js/routes/factories/network.js

```diff
@@ -1,5 +1,7 @@
 /* eslint-disable no-unused-vars */
-import React, { PropTypes } from "react";
+import PropTypes from "prop-types";
```

Comment:
I think we shouldn't have PropTypes inside the eslint-disable, but that is no blocker for me, we can adjust afterwards 👍